scripts/jobrunner.sh

```diff
@@ -80,6 +82,9 @@ test -f "$playbook" && ln -sf "$playbook" ~jobrunner/playbook.md
 
 # ensure file ownership and permissions
 chown -R jobrunner:jobrunner /srv/jobrunner
+chmod 0600 $secrets_env
+chmod 0700 $DIR/secret
+find $DIR/secret -type f -exec chmod 0600 {} \;
```

Comment:
should this be `"$DIR"/secret ...`?

Reply:
hmm maybe not - it's not done everywhere, so...

Reply:
It should but why did the lint checks pass if so?

I authored these changes on the EMIS VM, where we currently can't install stuff (like shellcheck)

Reply:
narrator: it was because the linter spotted that the var `DIR` did not have any spaces, so did not need to be quoted